### PR TITLE
#185106772 Google auth

### PR DESCRIPTION
**Pull request by Jmukakalisa**
**What does this PR do?**

- This PR implements the Google signin button feature

**Description of Task to be completed?**

- Users should be able to signing with google account

**How should this be manually tested?**

- Clone this project run the script "npm run dev" then go to signin

**Any background context you want to provide?**
N/A
**What are the relevant pivotal tracker/Trello stories?**

- Tracker: 185106772

**Screenshot**
Screenshot 2023-05-30 at 12 11 33

**Questions**

- None

### CODE REVIEW

**Review by kirengaboris — APPROVED** [review]

**Review by masterchief-00 — APPROVED** [review]

File: src/main.jsx [hunk 1]
```diff
@@ -11,7 +10,6 @@ ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
     <Provider store={store}>
       <App />
-      <ToastContainer/>
     </Provider>
```
Comment (fab-ryan) [inline src/main.jsx]: why did you remove ToastContainer???

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Check on comments

File: src/main.jsx (hunk 1 above)
Comment (kirengaboris) [reply src/main.jsx]: it is already in the login and signup pages adding it here displayed it twice

**Review by fab-ryan — APPROVED** [review]
